tests/installation/ntp_configuration.pm

```diff
@@ -0,0 +1,22 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: This test module launches the installation from
+#          the installation settings page.
+
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use strict;
+use warnings;
+use base 'y2_installbase';
+use Test::Assert ':all';
+
+sub run {
+    my $ntp_settings = $testapi::distri->get_ntp_config()->get_ntp_config();
```

Comment:
I would recommend to give different names to these functions. Like get_ntp_value().

Reply:
please use names in the screens, `get_npt_servers` in the test layer.